### ISSUE TITLE
Add chocolateygui_config and chococlateygui_feature Modules

### ISSUE THREAD

**Issue by ryanrichter94** [chocolatey/chocolatey-ansible#66]
@pauby @gep13 
Would like to raise the question if this Ansible collection would be the correct place to request a win_chocolateygui_config and win_chocolateygui_feature ansible module. 

Would like to be able to manage the config and features available within Chocolatey GUI in much the same was as the current win_chocolatey_config and win_chocolatey_feature modules work within this current collection. 

Know it is currently possible to make these changes via powershell running a `chocolateyguicli feature` or `chocolateyguicli config` command.

**Comment by vexx32** [comment]
Yeah, could do. You could likely lift the majority of the current win_chocolatey_feature and win_chocolatey_config modules, replace a few executable names, check args syntax and config file paths and error messages, and have it relatively ready to go.

I'd suggest naming the new modules `win_chocolateygui_config` and `win_chocolateygui_feature` for consistency's sake.

**Comment by pauby** [comment]
Great suggestion @ryanrichter94 ! I think adding this makes sense.
